script.js

```diff
@@ -0,0 +1,318 @@
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].EN+'</div>';
+                    }else{
+                        out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].en+'</div>';
+                    }
+                }
+                el.innerHTML = out;
+            });
+        }
+        if(key == 'en'){
+            let rows = document.querySelectorAll('.keyboard__row');
+            rows.forEach((el,i) => {
+                let out = '';
+                for(let j = 0; j < ArrKey[i].length; j++){
+                    out+='<div class ="k-key '+ArrKey[i][j].code+'">'+ArrKey[i][j].en+'</div>';
+                }
+                el.innerHTML = out;
+            });
+        }
+    }
+}
+
+
+PrintContent();
+
+PrintKeyboard(languageFlag, 0);
+
+
+
+//переключение языков и размера шрифта
+document.addEventListener('keydown',function(evt){
+    if(evt.shiftKey && evt.altKey){
+        switch(languageFlag){
+            case 'en': 
+                languageFlag ='ru';
+                break;
+            case 'EN':
+                languageFlag = 'RU'; 
+                break;
+            case 'RU': 
+                languageFlag = 'EN'; 
+                break;
+            case 'ru': 
+                languageFlag ='en'; 
+                break;
+        }
+        PrintKeyboard(languageFlag, false);
+    }else if(evt.code == 'CapsLock'){
+        switch(languageFlag){
+            case 'en': 
+                languageFlag ='EN';
+                break;
+            case 'EN':
+                languageFlag = 'en'; 
+                break;
+            case 'RU': 
+                languageFlag = 'ru'; 
+                break;
+            case 'ru': 
+                languageFlag ='RU'; 
+                break;
+        }
+        PrintKeyboard(languageFlag, false);
+    }else if(evt.shiftKey){
+        PrintKeyboard(languageFlag, true);
+    }
+    localStorage.setItem('languageFlag', languageFlag);
+});
+
+
+
+
+// подсвечивание кнопок при нажатии
+    let first = document.querySelector('.keyboard');
+    document.addEventListener('keydown',function(evt){
+        let rows = document.querySelectorAll('.keyboard__row');
+        for(let i = 0; i < ArrKey.length; i++){
+            for(let j= 0; j < ArrKey[i].length; j++){
+                if(evt.code === ArrKey[i][j].code){
+                    let pressButton = document.querySelector("."+ ArrKey[i][j].code);
+                    pressButton.classList.add('press');
+                    
+                }
+            }
+        }
+    });
+// удвление подсветки при отжатии
+    document.addEventListener('keyup',function(evt){
+        for(let i = 0; i < ArrKey.length; i++){
+            for(let j = 0; j < ArrKey[i].length; j++){
+                if(evt.code === ArrKey[i][j].code){
+                    let pressButton = document.querySelector("."+ ArrKey[i][j].code);
+                    pressButton.classList.remove('press');

```

Comment:
perhaps `classlist.toggle` can help you create one callback both for `keydown` & `keyup` events